genesis/engine/entities/avatar_entity/avatar_entity.py

```diff
@@ -9,16 +9,16 @@
 class AvatarEntity(RigidEntity):
     def add_link(
         self,
-        name,
+        name: str,
         pos,
         quat,
         inertial_pos,
         inertial_quat,
         inertial_i,
-        inertial_mass,
-        parent_idx,
-        invweight,
-    ):
+        inertial_mass: float,
+        parent_idx: int,
+        invweight: float,
```

Comment:
Are you sure this is float? It should be a vector of 2 elements actually :/

Reply:
Oh thanks! Updated. How does it look now?

Reply:
great !